test/accessibility/config.ts

```diff
@@ -17,15 +17,19 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     ...functionalConfig.getAll(),
 
     testFiles: [
-      require.resolve('./apps/discover'),
+      // these 5 tests all load addSampleDataSet('flights')
+      // only the last test does removeSampleDataSet('flights')
```

Comment:
NOTE: I started going down the path to create an index file with a before and after to load/unload the sample data for this group of tests but the last test `kibana_overview` appears to have tests for both states where there is and isn't data.  I need to talk to @bhavyarm about refactoring these.  But I'd like to get the test unskipped in any case.